docs/field-configurations.md

```diff
@@ -9,13 +9,53 @@ They must be added to a file named `configs.yml` in the assets template folder o
 The config file is a yaml file consisting of an array of config entry.
 
 For each config entry the following fields are available:
-- `name` *mandatory*: dotted path field, as in `fields.yml`
```

Comment:
why removing "as in `fields.yml`", it's relevant that the fields defined in the config file must match an entry in the fields.yml, otherwise they will be ignored :)